### ISSUE TITLE
Incorrect register addresses in stm32wb55xx.h

### ISSUE THREAD

**Issue by kasjer** [STMicroelectronics/cmsis_device_wb#2]
**Describe the bug**
Following section found in **stm32wb55xx.h** file is probably there by accident.
Addresses belong to I2C3 and CRS peripherals not USB
Correct base address for packet memory can be also found in this header defined as **USB1_PMAADDR**,
same for **USB1_BASE**

```c
#define USB_BASE                              (0x40005C00UL)                   /*!< USB_IP Peripheral Registers base address */
#define USB_PMAADDR_Pos           (13U)                                        
#define USB_PMAADDR_Msk           (0x20003UL << USB_PMAADDR_Pos)               /*!< 0x40006000 */
#define USB_PMAADDR               USB_PMAADDR_Msk                              /*!< USB_IP Packet Memory Area base address */
                                             
#define USB_CNTR                             (USB_BASE + 0x40)           /*!< Control register */
#define USB_ISTR                             (USB_BASE + 0x44)           /*!< Interrupt status register */
#define USB_FNR                              (USB_BASE + 0x48)           /*!< Frame number register */
#define USB_DADDR                            (USB_BASE + 0x4C)           /*!< Device address register */
#define USB_BTABLE                           (USB_BASE + 0x50)           /*!< Buffer Table address register */
#define USB_LPMCSR                           (USB_BASE + 0x54)           /*!< LPM Control and Status register */
#define USB_BCDR                             (USB_BASE + 0x58)           /*!< Battery Charging detector register*/
```

**Comment by ASELSTM** [comment]
ST Internal Reference: 126136

**Comment by ASELSTM** [comment]
Hi @kasjer,

The issue you reported has been fixed in the frame of version **v1.12.0** of the **cmsis_device_wb** recently published on GitHub.
Please allow me thus to close this thread.

With regards,
